package.json

```diff
@@ -50,8 +50,8 @@
     "changeset"
   ],
   "dependencies": {
-    "ember-changeset": "0.13.3",
-    "ember-changeset-validations": "^0.9.3",
+    "ember-changeset": "^1.0.0",
```

Comment:
I might be wrong, but you could potentially remove this from your deps -- ECV will bring it in and use the version it was tested against. Less chance of using an incompatible version